external/net_http.patch

```diff
@@ -27,12 +27,20 @@ index 36c925a..86a5ba9 100644
    // By default libevents only allow GET, POST, HEAD, PUT, DELETE request
    // we have to manually turn OPTIONS and PATCH flag on documentation:
    // (http://www.wangafu.net/~nickm/libevent-2.0/doxygen/html/http_8h.html)
+   evhttp_set_allowed_methods(
+       ev_http_, EVHTTP_REQ_GET | EVHTTP_REQ_POST | EVHTTP_REQ_HEAD |
+                     EVHTTP_REQ_PUT | EVHTTP_REQ_DELETE | EVHTTP_REQ_OPTIONS |
+-                    EVHTTP_REQ_PATCH);
```

Comment:
why patch is removed?

Reply:
Patch is not removed, check final file after the change